src/App.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="flex">
     <Navbar/>
-    <main class="ml-24">
+    <main class="ml-24 pr-8 pl-8 pt-11 pb-11">
```

Comment:
Plutôt utiliser les class px et py
```suggestion
    <main class="ml-24 px-8 py-11">
```